### ISSUE TITLE
Too many downloads of probability maps when switching on other populations

### ISSUE THREAD

**Issue by zsinnema** [MetaCell/salk-interactive-atlas#116]
When switching on a second, third etc population and the density map switch (overlay) is active and a probability map is active then all probability maps of all the switched on populations are loaded.

It would be better (performance wise) to only load the one that is not already loaded

**Comment by zsinnema** [comment]
btw same when switching off a population, this should even not load anything.....

**Comment by afonsobspinto** [comment]
Yeah I haven't consider any optimizations on the requests because I thought the browser would cache the get requests so it wouldn't be a problem.
I will add logic to keep track of what's loaded and infer what's missing from it
